source/includes/_changes.md

```diff
@@ -0,0 +1,144 @@
+# Changes
+
+The changes of a dataset describe subsequent changes of states that affected the different sections of a dataset. Every action taken on any resource through POST, PUT or DELETE creates a change object that can be retrieved and acted upon.
+
+## The change object
+
+> An example change object
+
+```json
+{
+    "change_uid": 125,
+    "dataset": {
+        "dataset_uid": "dataset"
+        "domain": "domain"
+    },
+    "user": {
+        "username": "username"
+    },
+    "timestamp": "2017-06-07T15:16:05.701266+00:00",
+    "diff": {
+        "metadata": [
+            "path": ["default", "description"],
+            "old_value": null
+            "new_value": "dataset title"
+            "operation_type": create
+        ]
+    },
+    "sections": [
+        "metadata"
+    ]
+}
+```
+
+The change object contains:
+
+* a unique identifier
+* a reference to the dataset that changed
+* a reference to the user who created the new state
+* the day and time at which the change occured
+* the difference between the old and the new state of the dataset
+* the sections in which the change happened
+
+### Attributes
+
+Attribute | Description
+--------- | -----------
+`change_uid` <br> *string* | unique identifier for the change
+`dataset` <br> [dataset object](#the-dataset-object) | dataset targeted by the change <br> *expandable*
+`user` <br> [user object](#the-user-object) | user who made the change <br> *expandable*
+`timestamp` <br> *string* | time at which the change was made
+`diff` <br> *string* | difference between the state before and after the change
+`sections` <br> *array* | sections modified by this change
+
+## List all changes
+
+This endpoint lists all changes made to a dataset.
+
+> Definition
+
+```HTTP
+GET https://{DOMAIN_ID}.opendatasoft.com/api/management/v2/datasets/{DATASET_ID}/changes/
+```
+
+
+### Parameters
+
+Parameter | Description
+--------- | -----------
+dataset_uid <br> *string* | identifier of the dataset whose changes are to be listed
+
+Returns a list of changes for this datasets.
+
+```shell
+curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/changed_dataset/changes/ \
+    -u username:password
+```
+
+> Example response
+
+```json
+[
+    {...},
+    {
+        "change_uid": 126,
+        "dataset": {
```

Comment:
It's not expanded, the domain is part of the key. It's the other instance that is wrong :-)